AsyncDisplayKit/ASCellNode.m

```diff
@@ -132,7 +122,7 @@ - (void)setText:(NSString *)text
   _textNode.attributedString = [[NSAttributedString alloc] initWithString:_text
                                                                attributes:@{NSFontAttributeName: [UIFont systemFontOfSize:kFontSize]}];
 
-  [self invalidateCalculatedSize];
+  [self invalidateCalculatedLayout];
```

Comment:
`-invalidateLayout` could be a better (and shorter) name. But then we also need to think about renaming `calculatedLayout` property, since `-invalidateLayout` doesn't clearly imply that the `calculatedLayout` will be invalidated. Should we change `calculatedLayout` to `layout`?

I think it's worth investigating re-layout handling. Definitely an interesting follow-up ticket.